kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
+  labels:
+    app: celery-beat
```

Comment:
[Importance 6/10]

We should dod this trick https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments

Reply:
I think this is done now, thanks.